test/utility/parse_opts.c

```diff
@@ -27,6 +27,15 @@
  */
 #include "test_util.h"
 
+static void
+print_cmd_line_args(int argc, char * const *argv)
```

Comment:
@Chenhao-Huang, this function doesn't belong in `parse_opts.c`: can you please rename it `testutil_print_cmdline` and move it to `test/utility/misc.c`?  It will then also need a declaration in `test_util.h`.

Reply:
@michaelcahill I have modified the code as required.